src/SignalR/clients/ts/FunctionalTests/scripts/run-tests.ts

```diff
@@ -2,6 +2,7 @@ import { ChildProcess, exec, spawn } from "child_process";
 import { EOL } from "os";
 import { Readable } from "stream";
 
+import * as os from "os";
```

Comment:
Is this used or just an artifact of making these changes?

Reply:
Old code. Will remove